File: CPP/Day6/person.cpp
#include<iostream>
using namespace std;

class person
{
    protected :

    int Age;
    string name;

    public:

        person()
        {
            Age = 24;
            name = "prasad";
        }

        person(int a , string nm)
        {
            Age = a;
            name = nm;
        }

        ~person()
        {

        }

       virtual char Display() = 0;
        


};

class Student : public person
{
    char gread;

    public :

    Student()
    {
        gread = 'C';
    }

    Student( char g): person()
    {
        gread = g;
    }


    Student(int a , string nm, char g): person(a,nm)
    {
        gread = g;
    }

    ~Student()
    {

    }



    char Display()
    {
        cout<<"Student Information =>> "<<Age<<" "<<name<<endl;
        return gread;
    }

};



int main()
{
    cout<<"\n======================[ obj-1 ]=========================\n"<<endl;
    person *p1 = new Student(25,"sagar",'A');

    cout<<"Student Pass With Gread ==>>  [ "<<p1->Display()<<" ]";
    delete p1;
    cout<<"\n=========================================================\n"<<endl;

    cout<<"\n======================[ obj-2 ]=========================\n"<<endl;
    person *p2 = new Student('A');

    cout<<"Student Pass With Gread ==>>  [ "<<p2->Display()<<" ]";
    delete p2;
    cout<<"\n=========================================================\n"<<endl;

    cout<<"\n======================[ obj-3 ]=========================\n"<<endl;
    person *p3 = new Student(23,"Tejas",'B');

    cout<<"Student Pass With Gread ==>>  [ "<<p3->Display()<<" ]";
    delete p3;
    cout<<"\n=========================================================\n"<<endl;





    return 0;
}

File: Core-Java/Assignment/Assignment_01/src/Person.java

public class Person 
{
	
	private String name;
	private int age;
	
	public Person()
	{
		age=18;
	}
	
	public Person(String n)
	{
		name=n;
		age=18;
	}
	
	public Person(String n, int a)
	{
		name=n;
		age=a;
	}
	
	public void Display()
	{
		System.out.println("Name is "+name+"  & Age is "+age);
	}
	
	public static void main(String[] args) 
	{
		Person p1 = new Person();
		p1.Display();
		
		Person p2 = new Person("Prasad");
		p2.Display();
		
		Person p3 = new Person("Sagar" , 24);
		p3.Display();
	}
	
}
